packages/fx-core/src/question/inputs/CreateSampleProjectInputs.ts

```diff
@@ -0,0 +1,31 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+
+import { Inputs } from "@microsoft/teamsfx-api";
+
+export interface CreateSampleProjectInputs extends Inputs {
```

Comment:
Will sample owner need to update here if new sample onboarded? cc @tecton

Reply:
This file is automatically generated by our prebuild script. Sample owner don't need to change it manually. But he will be aware of the change once the sample choices are changed. 
I will add some comments on this file.